apps/bees/src/op.c

```diff
@@ -69,6 +70,7 @@ const op_id_t userOpTypes[NUM_USER_OP_TYPES] = {
   eOpPoly,
   eOpRandom,
   eOpRoute,
+  eOpRoute2
```

Comment:
this missing comma breaks compilation

Reply:
Stupid mistake, sorry about that... It is my first time in here (and I am well aware I am replying to this two months later), and instead of copying my code I rewrote it into github, hence the errors. Now how should this work? I am fine with correcting these of course, but should I do the changes myself or should I simply accept yours?